seacrowd/sea_datasets/vitext2sql/vitext2sql.py

```diff
@@ -0,0 +1,175 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@inproceedings{nguyen2020vitext2sql,
+    title      = {{A Pilot Study of Text-to-SQL Semantic Parsing for Vietnamese}},
+    author     = {Anh Tuan Nguyen and Mai Hoang Dao and Dat Quoc Nguyen},
+    booktitle  = {Findings of the Association for Computational Linguistics: EMNLP 2020},
+    year       = {2020},
+    pages      = {4079--4085}
+}
+"""
+
+_DATASETNAME = "vitext2sql"
+
+_DESCRIPTION = """\
+This is the first public large-scale Text-to-SQL semantic parsing dataset for Vietnamese.
+The dataset is created by manually translating the Spider dataset into Vietnamese.
+"""
+
+_HOMEPAGE = "https://github.com/VinAIResearch/ViText2SQL"
+
+_LICENSE = f"""{Licenses.OTHERS.value} |
+By downloading the ViText2SQL dataset, USER agrees:
+1. to use ViText2SQL for research or educational purposes only.
+2. to not distribute ViText2SQL or part of ViText2SQL in any original or modified form.
+3. and to cite our EMNLP-2020 Findings paper above whenever ViText2SQL is employed to help produce published results.
+Copyright (c) 2020 VinAI Research
+THE DATA IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE DATA OR THE USE OR OTHER DEALINGS IN THE
+DATA.
+"""
+
+_SOURCE_VERSION = "1.0.0"
+
+_URLS = {
+    "word-level": {
+        "train": "https://raw.githubusercontent.com/VinAIResearch/ViText2SQL/master/data/word-level/train.json",
+        "test": "https://raw.githubusercontent.com/VinAIResearch/ViText2SQL/master/data/word-level/test.json",
+        "validation": "https://raw.githubusercontent.com/VinAIResearch/ViText2SQL/master/data/word-level/dev.json",
+    },
+    "syllable-level": {
+        "train": "https://raw.githubusercontent.com/VinAIResearch/ViText2SQL/master/data/syllable-level/train.json",
+        "test": "https://raw.githubusercontent.com/VinAIResearch/ViText2SQL/master/data/syllable-level/test.json",
+        "validation": "https://raw.githubusercontent.com/VinAIResearch/ViText2SQL/master/data/syllable-level/dev.json",
+    },
+}
+
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class Vitext2sql(datasets.GeneratorBasedBuilder):
```

Comment:
Hi @williamnixon20, since this one is minor, I'll add it directly via `Commit Suggestion` feature, no need to make any changes from your side.

btw @jamesjaya , wanna do another test for this new implementation?